### PR TITLE
Make LoadingIndicator more general and use it more consistently across owid

### PR DESCRIPTION
**Pull request by MarcelGerber**
Currently, loading charts have "pulsating" loading indicator, while the lightbox and download tab use `fa-spinner`.

This changes `LoadingIndicator` to use the pulsating one and uses it for loading charts, the lightbox and the download tab.

### Lightbox
image
image

### Download tab
image
image

This also makes it easier to re-use the element in the future.

### CODE REVIEW

File: charts/client/chart.scss [hunk 1]
```diff
@@ -14,15 +14,13 @@ $zindex-Tooltip: 20;
 $zindex-EntitySelector: 30;
 $zindex-IndicatorDropdown: 50;
 
-figure[data-grapher-src]:empty,
-.loading-chart {
+figure[data-grapher-src]:empty {
```
Comment (MarcelGerber) [inline charts/client/chart.scss:17]: I wasn't really sure in which case `figure[data-grapher-src]:empty` applies, if any.
If it isn't being used we can remove these 30 lines here, but I didn't wanna break anything just yet.

Comment (danielgavrilov) [reply charts/client/chart.scss:17]: It applies while JavaScript is loading on a `/grapher/` page.

Comment (MarcelGerber) [reply charts/client/chart.scss:17]: Ah, makes sense.
I don't like the duplication we have here, but can't really come up with a nicer solution either (except for changing the HTML of the `/grapher` pages, but that's not nice either).

Maybe a CSS mixin for the loading indicator?

Comment (MarcelGerber) [reply charts/client/chart.scss]: Okay, I've now gone with including them server-side.
The LoadingIndicator is then replaced with the interactive chart once React renders it.

**Comment by danielgavrilov** [comment]
Nice to have this component standardized everywhere!

**Pull request merged**